stubs/WordPress/functions.php

```diff
@@ -1610,15 +1610,15 @@ function wp_rand( $min = 0, $max = 0 ) {
 }
 
 /**
- * @param string           $handle
- * @param string|bool      $src
- * @param array<string>    $deps
- * @param string|bool|null $ver
- * @param bool             $in_footer
+ * @param string                                $handle
+ * @param string|bool                           $src
+ * @param array<string>                         $deps
+ * @param string|bool|null                      $ver
+ * @param array{0: string, 1: string|bool}|bool $args
```

Comment:
I think the correct way to do this would be

```php
/**
  * @param bool|array{in_footer?: bool, strategy?: string} $ args {
  *     @type bool $in_footer
  *     @type string $strategy
  * }
  */
```

Unfortunately, the array properties have to be there twice so they get picked up by all tools :/